### PR TITLE
Add elastic stresses to surface stress output

### PR DESCRIPTION
**Pull request by anne-glerum**
In line with the normal stress visualization postprocessor, the surface stress PP now includes the elastic stresses as well. The names of the stress tensor components are also updated to avoid running into duplicate paraview output names and to differentiate between viscous and visco-elastic stresses. 

### For all pull requests:

* [x] I have followed the [instructions for indenting my code](../blob/master/CONTRIBUTING.md#making-aspect-better).

### For new features/models or changes of existing features:

* [ ] I have tested my new feature locally to ensure it is correct.
* [ ] I have [created a testcase](http://www.math.clemson.edu/~heister/manual.pdf#sec%3Awriting_tests) for the new feature/benchmark in the [tests/](../blob/master/tests/) directory.
* [ ] I have added a changelog entry in the [doc/modules/changes](../blob/master/doc/modules/changes) directory that will inform other users of my change.

### CODE REVIEW

**Comment by anne-glerum** [comment]
The test fails because of the new names. There is no test with elasticity turned on.

**Review by gassmoeller — COMMENTED** [review]
Looks good except for my one question about the name.

**Comment by gassmoeller** [comment]
Oh also please update the test output when we have decided about the name.

File: source/postprocess/visualization/surface_stress.cc [hunk 1]
```diff
@@ -83,25 +99,31 @@ namespace aspect
       SurfaceStress<dim>::get_names () const
       {
         std::vector<std::string> names;
-        switch (dim)
+        if (this->get_parameters().enable_elasticity == true)
```
[the remark this answers is not in the document]
Comment (anne-glerum) [reply source/postprocess/visualization/surface_stress.cc]: Sure, I'll rename to just `surface_stress_`.

**Review by gassmoeller — APPROVED** [review]

**Comment by anne-glerum** [comment]
> Oh also please update the test output when we have decided about the name.

Done :)

**Pull request merged**